src/context.rs

```diff
@@ -351,15 +351,22 @@ impl<'buf> Secp256k1<AllPreallocated<'buf>> {
 
     /// Creates a context from a raw context.
     ///
+    /// The returned [`core::mem::ManuallyDrop`] context will never deallocate the memory pointed to
+    /// by `raw_ctx`, nor destroy the context, this may lead to memory leaks. `ManuallyDrop::drop`
+    /// (or [`core::ptr::drop_in_place`]) will only destroy the context, the caller is required to
```

Comment:
I used a semicolon (because it makes me feel like an English grammar guru). We now have:
```
    /// The returned [`core::mem::ManuallyDrop`] context will never deallocate the memory pointed to
    /// by `raw_ctx` nor destroy the context. This may lead to memory leaks. `ManuallyDrop::drop`
    /// (or [`core::ptr::drop_in_place`]) will only destroy the context; the caller is required to
    /// free the memory.
```